web/views/role_grant_view.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Cog.V1.RoleGrantView do
+  use Cog.Web, :view
+
+  def render("show.json", %{role: role}) do
+    %{role: %{id: role.id,
+              name: role.name,
+              permissions: render_permissions(role.permissions)}}
+  end
+
+  def render("index.json", %{roles: roles}) do
+    %{roles: Enum.map(roles, fn(role) ->
+        %{id: role.id,
+          name: role.name,
+          permissions: render_permissions(role.permissions)}
+    end)}
+  end
+
+  defp render_permissions(permissions) do
+    Enum.map(permissions, fn(permission) ->
+        %{id: permission.id,
+          name: permission.name,
+          namespace: permission.namespace.name}
+    end)
+  end
+
+end
```

Comment:
I wanted to set this such that it would use the same views as RoleView. Any ideas? The code is identical.